helm/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -122,6 +122,12 @@ spec:
         {{- if kindIs "bool" .Values.disableIngressGroupNameAnnotation }}
         - --disable-ingress-group-name-annotation={{ .Values.disableIngressGroupNameAnnotation }}
         {{- end }}
+        {{- if kindIs "bool" .Values.tolerateNonExistentBackendService }}
```

Comment:
Could you also add an entry in the `values.yaml`?

Reply:
Yep, updated.